pkg/scaffold/resources/deployment.go

```diff
@@ -6,6 +6,18 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
```

Comment:
sweet comments

Reply:
yeah! thought it would be nice not having to lookup docs everytime we tweak them